src/CollAction/Views/Projects/Find.cshtml

```diff
@@ -9,118 +9,7 @@
     var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<h2>@Localizer["Find"]</h2>
```

Comment:
Why don't I just roll the template file back that way the page will stay functional. I can leave in the stubbed out react components in the frontend application and not load them in to the page. That way we can cut back over when Mina or I are ready to start work on the find project page.